brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Toolbar options in the Inspector Sidebar.

### Description

- Toolbar options should also be present in the Inspector Sidebar.
- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/a3ca6973-d093-41dd-9e23-4f746fe4552f?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes
- Updated toolbar component where needed.

### Screenshots


### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Drag and drop UAG block.
- Check the toolbar alignment option or similar option it should be present in Inspector Sidebar Setting.
- For eg. See here https://share.getcloudapp.com/8Lup7p6Y

### Checklist:
